### DISCUSSION TITLE
Commit Message on Save

### DISCUSSION THREAD

**Discussion by fnafox** [requarks/wiki#5470] [General]
So I haven't seen this brought up anywhere, but something that I'd like to see is an option to add commit messages on save. In group collaborative environments, I find it would be useful to allow users to append messages when they make changes to a document akin to `git commit -m`, which would make browsing change history much easier.

It could be an optional functionality, but I think this option would significantly help managing large environments, especially if there was a toggle to additionally make it mandatory as desired.

**Comment by NGPixel** [comment]
See https://requarks.canny.io/wiki/p/custom-commit-message-custom-edit-summary
